jupyter_book/book_template/content/features/notebooks.ipynb

```diff
@@ -491,11 +491,69 @@
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": null,
+   "cell_type": "markdown",
    "metadata": {},
-   "outputs": [],
-   "source": []
+   "source": [
+    "## Executing notebooks\n",
+    "\n",
+    "✨**experimental**✨\n",
+    "\n",
+    "By default, Jupyter Book will not run any of your notebooks when you build your\n",
+    "book. It assumes you have already run each one, and simply converts each page to\n",
+    "HTML.\n",
+    "\n",
+    "However, sometimes you'd like to store *just* the book content, and then *run* this content\n",
+    "when you build your book. This helps you ensure that your content still runs, and\n",
+    "that you've always got the latest outputs in your book. The drawback to this is that\n",
+    "your book will take longer to build, especially if your content takes a long time\n",
+    "to run."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Note:** If you store your Notebook content as [Jupytext documents](jupytext.html), then\n",
```

Comment:
Maybe:

**Note:** If you store your Notebook content as [Jupytext documents](jupytext.html), such as `.md`, `.py` ..?? documents, then ...